src/App.tsx

```diff
@@ -10,33 +11,24 @@ export const tabs = [
 ];
 
 export const App: React.FC = () => {
-  return (
-    <div className="section">
-      <h1 className="title">
-        Selected tab is Tab 1
-      </h1>
-
-      <div data-cy="TabsComponent">
-        <div className="tabs is-boxed">
-          <ul>
-            <li className="is-active" data-cy="Tab">
-              <a href="#tab-1" data-cy="TabLink">Tab 1</a>
-            </li>
+  const [selectedTabId, setSelectedTabId] = useState('tab-1');
+  const [selectedTabTitle, setSelectedTabTitle] = useState('Tab 1');
+  const [selectedTabContent, setSelectedTabContent] = useState('Some text 1');
```

Comment:
Keep only one state, the selectedTabId and use the find method to save the selectedTab based on the id

Reply:
I implemented using selectedTab as a state. To avoid calling find in the runtime

Reply:
True I agree, I missunderstood the instructions, I thought they wanted us to use only one state.